projects/igniteui-angular/ng-package.json

```diff
@@ -12,6 +12,7 @@
     "igniteui-trial-watermark",
     "lodash-es",
     "uuid",
-    "@igniteui/material-icons-extended"
+    "@igniteui/material-icons-extended",
+    "igniteui-theming"
```

Comment:
Is the `ng-package.prod.json` used?

Reply:
My fault 🤦‍♂️ Going too fast for my own good.